boucle.py

```diff
@@ -163,24 +183,27 @@ def my_callback(frames):
 
     return jack.CALL_AGAIN
 
+
 client.set_process_callback(my_callback)
 
 # activate !
 with client:
-
     # make connection
     playback = client.get_ports(is_physical=True, is_input=True)
     if not playback:
         raise RuntimeError("No physical playback ports")
 
-    client.connect(outL, playback[0])
-    client.connect(outR, playback[1])
-
     record = client.get_ports(is_physical=True, is_output=True)
     if not record:
         raise RuntimeError("No physical record ports")
 
-    client.connect(record[0], inL)
-    client.connect(record[1], inR)
+    # if user has other connections that standard Main Out (i.e complex connections with dedicated outputs),
+    # then it is much better to leave the user creates his connections with jack patchbay instead of connecting automatically here.
+    if gui.auto_connect:
```

Comment:
no cms for the moment, you can chosse one, I will install it or you can install it yourself if you want.